go.mod

```diff
@@ -4,7 +4,7 @@ go 1.21.0
 
 require (
 	github.com/IBM/sarama v1.41.1
-	github.com/cloudtrust/common-service/v2 v2.6.18
```

Comment:
Do not forget to merge, release common service first and then update the depencies of this branch